pom.xml

```diff
@@ -29,7 +29,7 @@
     <groupId>bio.overture</groupId>
     <artifactId>maestro</artifactId>
     <!-- from .mvn/maven.config -->
-    <version>${revision}${sha1}${changelist}</version>
```

Comment:
same as above the version is in a config file

Reply:
reverted